meta-xfstk/recipes-xfstk/xfstk/xfstk_1.8.3.bb

```diff
@@ -6,9 +6,9 @@ inherit cmake qmake5_paths
 
 BBCLASSEXTEND = "native"
 
-DEPENDS = "libxml2 libusb boost qtbase doxygen"
+DEPENDS = "libxml2-native libusb-native boost-native qtbase-native"
 
-SRC_URI = "git://github.com/Exoyds/xfstk.git;tag=1.8.3;protocol=https"
+SRC_URI = "git://github.com/Exoyds/xfstk.git;tag=1.8.4;protocol=https"
```

Comment:
You could use `tag="${PV}"` to make the recipe version match the upstream tag.

Reply:
That's a good idea! I've made the change, and also bumped the overall version up to match.